### DISCUSSION TITLE
Kaart met duiklocaties + relevante gegevens ontsluiten

### DISCUSSION THREAD

**Discussion by user4294** [Nelos-VZW/dives-2#96] [Ideas]
Niet meteen een kernfunctionaliteit voor Dives 2.0 zelf, maar wat zouden jullie ervan denken om de kaart met duiklocaties te ontsluiten via Dives of de Neloswebsite, waarbij relevante gegevens over de duiklocaties worden weergegeven, zoals bijv. de gemiddelde zichtbaarheid en de zichtbaarheid en temperatuur over de voorbije 24-48u? Vermoed dat dit een meerwaarde kan zijn en extra motiveert om die data te loggen :)

**Comment by ndsmyter** [comment]
Je bedoelt data zoals in DIVES 1.0 ?
BV: https://leden.nelos.be/duikplaats/PL-OS-6-64

Het is niet dat ik er tegen ben, maar ik vind het ook niet echt de kerntaak van DIVES om dat aan te bieden. Waarmee ik dus wil zeggen dat het niet op mijn lijstje staat voor DIVES 2.0. Mogelijks iets voor DIVES 2.2 eind volgende jaar ? (DIVES 2.1 zal voornamelijk ledenadministratie verbeteringen zijn).

In elk geval: de data waarover je spreekt kunnen we wel eenvoudig per duikplaats aanbieden, we moeten gewoon de tijd steken om dergelijke pagina's te maken om dat op te gaan raadplegen.

**Comment by user4294** [comment]
Schitterend, ik wist niet van het bestaan van die pagina's af. Zou heel mooi zijn als dit op een bepaald moment (opnieuw) mogelijk wordt.

**Comment by ndsmyter** [comment]
FYI: Als je een duikplaats wil selecteren voor een activiteit in DIVES 1.0, heb je daar rechts van een tekstje "info" staan per duikplaats. Dat is een link naar de pagina die ik hier ook gelinkt heb.
Elke duikplaats heeft zo een pagina.
Maar is niet alom bekend bij de duikers, zit misschien ook wel wat verstopt.
<img width="303" alt="image" src="https://user-images.githubusercontent.com/648669/193821016-7d808408-cdcc-4a4a-a004-ee0486478514.png">


Op zich is dat allemaal niet zo heel veel werk om zo een pagina te maken, dus lijkt me niet onwaarschijnlijk dat we dit ooit terug toevoegen. Staat momenteel gewoon niet bovenaan de lijst :-)
